test/e2e/common/pods.go

```diff
@@ -848,4 +853,115 @@ var _ = framework.KubeDescribe("Pods", func() {
 		validatePodReadiness(false)
 
 	})
+
+	ginkgo.It("should run through the lifecycle of Pods and PodStatus", func() {
+		podResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"}
+		testNs := "default"
+		testPodName := "pod-test"
```

Comment:
if this test runs in parallel with another test that uses the same name `pod-test` there will be a conflict.
ideally object names should have a UUID `pod-test-<uuid>`

Reply:
I'm not sure this is a problem, since the namespace has been fixed to be dynamic to the test in 1e98c1ba6c0.

Reply:
that is true.